libs/partners/ai21/tests/unit_tests/test_embeddings.py

```diff
@@ -0,0 +1,70 @@
+"""Test embedding model integration."""
+from unittest.mock import Mock
+
+import pytest
+from ai21 import AI21Client, MissingApiKeyError
+from ai21.models import EmbedResponse, EmbedResult, EmbedType
+from pytest_mock import MockerFixture
+
+from langchain_ai21.embeddings import AI21Embeddings
+from tests.unit_tests.conftest import DUMMY_API_KEY
+
+_EXAMPLE_EMBEDDING_0 = [1.0, 2.0, 3.0]
+_EXAMPLE_EMBEDDING_1 = [4.0, 5.0, 6.0]
+_EXAMPLE_EMBEDDING_2 = [7.0, 8.0, 9.0]
+
+_EXAMPLE_EMBEDDING_RESPONSE = EmbedResponse(
+    results=[
+        EmbedResult(_EXAMPLE_EMBEDDING_0),
+        EmbedResult(_EXAMPLE_EMBEDDING_1),
+        EmbedResult(_EXAMPLE_EMBEDDING_2),
+    ],
+    id="test_id",
+)
+
+
+@pytest.mark.requires("ai21")
+def test_initialization__when_no_api_key__should_raise_exception() -> None:
+    """Test integration initialization."""
```

Comment:
maybe add a `del os.environ["AI21_API_KEY"]` here for people locally running in an environment that sets it

Reply:
Done. Added a function which deletes and sets the api key if exists